addons/uom/data/uom_data.xml

```diff
@@ -1,205 +1,176 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo noupdate="1">
 
-    <!-- UOM Categories -->
-    <record id="product_uom_categ_unit" model="uom.category">
-        <field name="name">Unit</field>
-    </record>
-    <record id="product_uom_categ_kgm" model="uom.category">
-        <field name="name">Weight</field>
-    </record>
-    <record id="uom_categ_wtime" model="uom.category">
-        <field name="name">Working Time</field>
-    </record>
-    <record id="uom_categ_length" model="uom.category">
-        <field name="name">Length / Distance</field>
-    </record>
-    <record id="uom_categ_surface" model="uom.category">
-        <field name="name">Surface</field>
-    </record>
-    <record id="product_uom_categ_vol" model="uom.category">
-        <field name="name">Volume</field>
+    <!-- DECIMAL.PRECISION -->
+    <record forcecreate="True" id="decimal_product_uom" model="decimal.precision">
+        <field name="name">Product Unit of Measure</field>
+        <field name="digits" eval="2"/>
     </record>
 
     <!-- UOM.UOM -->
     <!-- Units -->
     <record id="product_uom_unit" model="uom.uom">
-        <field name="category_id" ref="product_uom_categ_unit"/>
         <field name="name">Units</field>
-        <field name="factor" eval="1.0"/>
-        <field name="uom_type">reference</field>
+        <field name="relative_factor" eval="1.0"/>
```

Comment:
I would use factor when there is no related unit. Just in this file because people will copy it

Reply:
factor is computed and readonly.